src/core/server/ui_settings/settings/theme.ts

```diff
@@ -34,6 +34,19 @@ import { UiSettingsParams } from '../../../types';
 
 export const getThemeSettings = (): Record<string, UiSettingsParams> => {
   return {
+    'theme:enableUserControl': {
+      name: i18n.translate('core.ui_settings.params.enableUserControlTitle', {
+        defaultMessage: 'Enable user control',
+      }),
+      value: true,
+      description: i18n.translate('core.ui_settings.params.enableUserControlText', {
+        defaultMessage: `Enable users to control theming and dark or light mode via "Appearance" control in top navigation. When true, those settings can no longer be set globally by administrators.`,
+      }),
```

Comment:
@kgcreative Some new copy to review. One thing to consider is the fact that all UI settings within the same group ("Appearance", in this case), will be listed alphabetically by the key `theme:enableUserControl`. Ideally, I think this setting would precede the other two `theme:` prefixed settings on the advanced settings page. So maybe just `enableUserControl` instead, even though it currently just controls theme-related settings?